src/clients/inst_setup_dhcp.rb

```diff
@@ -1,12 +1,24 @@
 require "yast"
 require "network/network_autoconfiguration"
 
-def main
-  Yast::NetworkAutoconfiguration.instance.configure_dhcp
+module Yast
+  class SetupDhcp
```

Comment:
This class should live in `network/clients` (see [inst_update_installer](https://github.com/yast/yast-installation/blob/master/src/lib/installation/clients/inst_update_installer.rb) as example.